Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -79,6 +79,11 @@ public bool IsActiveInstance
         /// <summary>
         /// The public property of the Active Profile, ensuring events are raised on the change of the configuration
         /// </summary>
+        /// <remarks>
+        /// When setting the ActiveProfile, all currently running services will be destroyed and those specified in
```

Comment:
Would be good to mention this in release notes.

Reply:
Will do! Good call

Reply:
This is not new behavior, but I will definitely add it to the what's new section.